deployment/ansible/roles/raster-foundry.jenkins/tasks/main.yml

```diff
@@ -0,0 +1,6 @@
+---
+# tasks for raster-foundry.jenkins
+
+- name: Add new Nginx config
+  template: src=jenkins.conf.j2 dest=/etc/nginx/sites-enabled/jenkins.conf
+  notify: Restart Jenkins
```

Comment:
Because this template is for an Nginx configuration file, we'd want to restart Nginx here, not Jenkins.